Sources_v3/StreamChatUI/Utils/UIView+Extensions.swift

```diff
@@ -46,6 +46,11 @@ extension UIView {
         translatesAutoresizingMaskIntoConstraints = false
         return self
     }
+
+    var isShown: Bool {
+        get { !isHidden }
+        set { isHidden = !newValue }
+    }
```

Comment:
nit: `isHidden` / `isVisible` ?

Reply:
fixed in e23d9960734d6cfb4bfdbeca8d892f55fb70eadc